src/client/components/Graph.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import {
+  CartesianGrid,
+  Legend,
+  Line,
+  LineChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis
+} from 'recharts'
+
+const graphTitle = {
+  marginLeft: '7rem'
+}
+
+interface GraphProps {
+  title: string
+  data: any[]
+  xDataKey: string
+  labelValue: string
+  lineDataKey1: string
+  lineDataKey2: string
+  lineDataKey3: string
+  lineDataKey4: string
+  color1: string
+  color2: string
+  color3: string
+  color4: string
+  name1: string
+  name2: string
+  name3: string
+  name4: string
```

Comment:
See if you can condense this into an array of objects. Each element in the array will correspond to a line on the graph, with all its information. This should then allow the `Graphs` component to be flexible for use with any amount of lines.